examples/demo/src/reviews/AcceptButton.tsx

```diff
@@ -25,14 +25,14 @@ const AcceptButton = () => {
         {
             mutationMode: 'undoable',
             onSuccess: () => {
-                notify('resources.reviews.notification.approved_success', {
+                notify('resources.reviews.notifications.approved_success', {
```

Comment:
As the new namespace for custom resource notifications is `resources.{resource}.notifications`, it made sense to use the same namespace in this demo.